api/validation/api_test.go

```diff
@@ -147,35 +148,32 @@ func TestImportFromKeycloak(t *testing.T) {
 	user.ExportToKeycloak(&kcUser)
 
 	var imported = UserRepresentation{}
-	imported.ImportFromKeycloak(&kcUser)
+	imported.ImportFromKeycloak(kcUser)
 
-	assert.Equal(t, user, imported)
```

Comment:
You are not testing other fields than BirthDate like in the previous version... Perhaps you could restore the initial test?
If both user.BirthDate and imported.BirthDate are not strictly equals, you can set them to nil before testing equality of user and imported